### DISCUSSION TITLE
Prefetch not working on `Link` since we removed `prefetch={true}`

### DISCUSSION THREAD

**Discussion by kasbah** [vercel/next.js#29496] [Help] [answered]
### What version of Next.js are you using?

11.1.1

### What version of Node.js are you using?

14

### What browser are you using?

Firefox, Chrome

### What operating system are you using?

Linux

### How are you deploying your application?

next start

### Describe the Bug

We removed `prefetch={true}` from our `Link` tags but it doesn't seem to be prefetching based on intersection observer as promised. 

### Expected Behavior

Prefetching should happen when cards come in to view. 

### To Reproduce

Visit https://master.staging.kitspace.dev and have a look at the network requests when scrolling. I can't observe any prefetch requests. 

The cards are using `Link` tags ([view source](https://github.com/kitspace/kitspace-v2/blob/63e8b6f71ce312bd5cc85fb9d090ff56ef9be5b1/frontend/src/components/ProjectCard.js#L22-L44)).

**Comment by ijjk** [comment, marked as answer]
Hi, it looks like you aren't setting an `a` tag as the direct child of `next/link` [here](https://github.com/kitspace/kitspace-v2/blob/63e8b6f71ce312bd5cc85fb9d090ff56ef9be5b1/frontend/src/components/ProjectCard.js#L23) causing the `ref` to not be handled correctly. We recommend always wrapping your link in an `a` tag for better accessibility and link handling.
